app/javascript/components/DropdownOption.jsx

```diff
@@ -52,6 +52,17 @@ class DropdownOption extends React.Component {
     this.props.onSave(optionId, text);
   }
 
+  getComponentStyle(currentValue, option) {
```

Comment:
I'm not too familiar with CSS standards but for the future I like the second approach [here](https://stackoverflow.com/questions/30533171/react-js-conditionally-applying-class-attributes)
So it would be something like 
```
var componentStyle = classNames(
  'input__dropdown',
  {
    'input__dropdown--selected': currentValue == option.id,
    'input__dropdown--disabled': !this.props.editable
  }
);
```
and then just return this variable